src/Symfony/Component/Security/Http/Authentication/AuthenticatorManager.php

```diff
@@ -62,12 +62,17 @@ public function __construct(
     }
 
     /**
-     * @param BadgeInterface[] $badges Optionally, pass some Passport badges to use for the manual login
+     * @param BadgeInterface[]     $badges Optionally, pass some Passport badges to use for the manual login
+     * @param array<string, mixed> $attributes Optionally, pass some Passport attributes to use for the manual login
```

Comment:
not sure why fabbot didn't pick this one
```suggestion
     * @param BadgeInterface[]     $badges     Optionally, pass some Passport badges to use for the manual login
     * @param array<string, mixed> $attributes Optionally, pass some Passport attributes to use for the manual login
```